templates/details_table.html

```diff
@@ -5,6 +5,26 @@
 {% block extra_details %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
+      {% if entity.custom_properties.security_classification == "Official-Sensitive"%}
+      <div class="govuk-body govuk-!-padding-top-7 govuk-!-padding-bottom-7">
```

Comment:
```suggestion
      <div class="govuk-body govuk-!-padding-top-7 govuk-!-padding-bottom-7">
```
I don't think this is actually body?

Reply:
No doesn't look like it. I've stuck govuk-body on in a couple of divs where we are wrapping markdown generated content but that's not the case here.